api/tests/unit/application/error-manager_test.js

```diff
@@ -103,4 +109,51 @@ describe('Unit | Application | ErrorManager', () => {
     });
   });
 
+  describe('#_mapToHttpError', () => {
+
+    it('should instantiate UnauthorizedError when MissingOrInvalidCredentialsError', async () => {
+
+      // given
+      const error = new MissingOrInvalidCredentialsError();
+      sinon.stub(HttpErrors, 'UnauthorizedError');
+
+      // when
+      const params = { request: {}, h: hFake, error };
```

Comment:
**Chipotage** : Dans le `// given` plutôt la constante ?

Reply:
What do you mean ?

Reply:
Corrigé